### ISSUE TITLE
Installed With UCS Traffic Monitor

### ISSUE THREAD

**Issue by ThomasMcDo** [paregupt/mds_traffic_monitor#7]
Is it possible to install this with the UCS Traffic Monitor deployment?

**Comment by paregupt** [comment]
Yes. The steps are the same. Some hints are in the readme. Let me know if you need help.

**Comment by ThomasMcDo** [comment]
Thanks for the reply. Yes I understand that, but can I run both on the same host/Grafana instance?

**Comment by paregupt** [comment]
Yes.

**Comment by ThomasMcDo** [comment]
Excellent.... I shall give it a try...... I appreciate the quick response.

**Comment by ThomasMcDo** [comment]
The local_sys.json do I merge that with the current one for UCS?

**Comment by paregupt** [comment]
Don't merge. Just use one of them. Prefer from UTM.

**Comment by ThomasMcDo** [comment]
I don't see the dashboards, I just copied them to /usr/local/UTM/grafana/dashboards (switches.json and switchports.json).

root@utm dashboards]# python3 /usr/local/telegraf/mds_traffic_monitor*.py -h
Traceback (most recent call last):
  File "/usr/local/telegraf/mds_traffic_monitor_high_frequency.py", line 17, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'

**Comment by paregupt** [comment]
Import dashbboards from grafana GUI. See this: https://www.since2k7.com/blog/2020/02/29/cisco-ucs-monitoring-using-grafana-influxdb-telegraf-utm-installation/#Import_dashboards

The other error is because Python module is missing, which shouldn't be the case with UTM VM because it has it installed. Did you install all the packages by yourself? Try installing requests module using pip3 install requests. Web search for more detials on installing Python modules.

**Comment by ThomasMcDo** [comment]
Installed the OVA and did a manual upgrade. The UCS part is working fine..

**Comment by paregupt** [comment]
ok,  I missed. UTM doesn't need requests module, whereas MTM does it need it because of NX-API. 
Install it via

`pip3 install requests`

Let me know how it goes. If you want, we can get on a webex to make it work.

**Comment by ThomasMcDo** [comment]
I guess this could also be extended to NX-OS based Cisco switches.

**Comment by paregupt** [comment]
Yes, it can be extended but need some development and testing effort.

BTW, did it work?

**Comment by ThomasMcDo** [comment]
Yes, thank you for the helpful hints.

**Comment by paregupt** [comment]
Glad to know. 

Anything you can share for helping others who do the same in the future? Installing the requests module is one thing. What else?

**Comment by ThomasMcDo** [comment]
I will upload a simple txt file with all the steps later today.... I am doing it again on a fresh install.. ;)

**Comment by paregupt** [comment]
Closing this older issue. Pls feel free to open a new issue if you still need help.
